src/wayland/data_device/server_dnd_grab.rs

```diff
@@ -212,6 +212,10 @@ where
         // we just forward the axis events as is
         handle.axis(details);
     }
+
+    fn start_data(&self) -> &GrabStartData {
+        unimplemented!()
+    }
```

Comment:
This will need to be implemented

Reply:
Where should I get the grab start data? The `start_dnd` function only accepts a serial.

Reply:
We can change the `start_dnd` function to take more arguments to fill out the grab data if needed.